### ISSUE TITLE
Improved settings flow

### ISSUE THREAD

**Issue by nblackburn** [msipos/syntaxic#21]
It would be nice if the settings window was organised into categories allowing for a better workflow and quicker access to commonly used settings.

**Comment by msipos** [comment]
I agree that it would be nice, but I find that it would be a lot of work.  I'd prefer to put this work in on adding features.  I've been planning to put in _a lot_ of preferences, as I am aware that programmers generally love to customize their workflow.

Perhaps a middle ground can be reached, where we can have some headers in the preferences dialog, and a list of most important settings up top?

**Comment by nblackburn** [comment]
It's not bad the way it is but the names would benefit being less programmatic and more human with perhaps a fuzzy search.

**Alternative**: You could take a leaf out of Sublimes book and turn it into a commented json file. Personally i love that about it but i am a geek at heart and get excited by terminals so you shouldn't trust me.

**Comment by msipos** [comment]
Actually, it used to be a JSON file.  I thought this GUI is a little better than a commented JSON file.

But you are absolutely right.  Will make the names more human, and add a fuzzy search by version 1.0
